src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,42 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MIN_USER_AGE = 18;
+    public static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        loginValidator(user);
+        ageValidator(user);
+        passwordValidator(user);
+        return storageDao.add(user);
+    }
+
+    private void passwordValidator(User user) {
+        if (user.getPassword() == null) {
+            throw new NullPointerException("Password can't be null");
+        }
```

Comment:
check [this message](https://mateacademy.slack.com/archives/C040S2B99FH/p1666701628636409)